.github/ISSUE_TEMPLATE/new_component.yaml

```diff
@@ -30,21 +38,24 @@ body:
   - type: textarea
     attributes:
       label: Telemetry data types supported
-      description: Any combination of traces, metrics and/or logs is valid here.
+      description: Any combination of traces, metrics, logs and/or profiles is valid here.
     validations:
       required: true
   - type: input
     attributes:
-      label: Code Owner(s)
-      description: A code owner is responsible for supporting the component, including triaging issues, reviewing PRs, and submitting bug fixes.
-        Please list one or more members or aspiring members of the OpenTelemetry project who will serve as code owners.
-        Having a code owner is strongly recommended. However, you may use the issue to try to find a code owner for your component.
+      label: Code Owners
+      description: Code owners are responsible for supporting the component, including triaging issues, reviewing PRs, and submitting bug fixes.
+        Please list three or more OpenTelemetry members who will serve as code owners.
+        One of them must be an approver or maintainer in this repository (it can be the same person as the sponsor).
+        Having at least two code owners before opening the issue is strongly recommended. However, you may use the issue to try to find code owners for your component.
```

Comment:
From above:
> Please list three or more OpenTelemetry members who will serve as code owners.

This line:

> Having at least two code owners before opening the issue is strongly recommended.

These two statements seem to be conflicting, or I'm missing something, can you clarify which it should be? (Maybe we're saying that between filing the issue and the component being accepted we'll get from the initial code owner count to three or more?)

Reply:
Maybe:
```
        Having at least two code owners before opening the issue is strongly recommended. However, you may use the issue to try to find enough code owners for your component to meet the minimum required number of three.
```
This may just end up being too wordy, it's not that unclear now that I've read it again. Feel free to drop.